tensorflow-gpu/Dockerfile

```diff
@@ -0,0 +1,98 @@
+FROM nvidia/cuda:8.0-cudnn5-devel
+MAINTAINER Nathan DeMaria <nathan.demaria@hudl.com>
+
+# Heavily based on https://github.com/tensorflow/tensorflow/commit/b3edccd6611e4ec365f0f9495a1913c0b4ba6416
+
+ENV CUDA_ROOT /usr/local/cuda/bin
+RUN apt-get update -y && apt-get install git wget vim libgtk2.0-0 -y
+
+# Miniconda/basic packages install
+RUN wget --quiet https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O install_anaconda.sh && \
+    /bin/bash install_anaconda.sh -b -p /opt/conda && \
+    /opt/conda/bin/conda update conda && \
+    # Set default python to 3.5
+    /opt/conda/bin/conda install python=3.5 && \
+    rm install_anaconda.sh
```

Comment:
Extremely minor, but should we rename this to `install_miniconda.sh` instead? Just to make it a tiny bit more clear.